pkgs/applications/science/robotics/emuflight-configurator/default.nix

```
@@ -0,0 +1,50 @@
+{lib, stdenv, fetchurl, unzip, makeDesktopItem, nwjs, wrapGAppsHook, gsettings-desktop-schemas, gtk3 }:
+
+let
+  pname = "emuflight-configurator";
+  desktopItem = makeDesktopItem {
+    name = pname;
+    exec = pname;
+    icon = pname;
+    comment = "Emuflight configuration tool";
+    desktopName = "Emuflight Configurator";
+    genericName = "Flight controller configuration tool";
+  };
+in
+stdenv.mkDerivation rec {
+  inherit pname;
+  version = "0.3.5";
+  src = fetchurl {
+    url = "https://github.com/emuflight/EmuConfigurator/releases/download/${version}/emuflight-configurator_${version}_linux64.zip";
+    sha256 = "d55bdc52cf93d58c728ccb296ef912a5fc0f42c57ed95f3ded5f85d1c10838c4";
+  };
+
+  nativeBuildInputs = [ wrapGAppsHook unzip ];
+
+  buildInputs = [ gsettings-desktop-schemas gtk3 ];
+
+  installPhase = ''
+    mkdir -p $out/bin \
+             $out/opt/${pname}
```

```suggestion
    mkdir -p $out/bin $out/share/${pname}
```
NixOS does not really use opt.

---

The opt folder needed by the nwjs wrapper which creates the final executable.

---

Those files should be under lib or share.

---

Moved to share